Bottom: 70bf4f0e3fc7280ac63a2fbe71ecc97d4858334a
Top:    e24e53e2449396ea426c0b9852c6ba92e40b5b2b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 22:35:12 +0000

Adjust STEP export to 0.2mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index d1b0452..7d1844a 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -31,7 +31,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 static int step_set_layer (const char *name, int group, int empty);
 static void use_gc (hidGC gc);
